pom.xml

```
@@ -149,7 +149,7 @@
         <maven-project-info-reports-plugin-version>3.0.0</maven-project-info-reports-plugin-version>
         <maven-surefire-report-plugin-version>${maven-surefire-plugin-version}</maven-surefire-report-plugin-version>
 
-        <!-- empty property used as placeholder for args injected by jacoco coverage plugin -->
+        <version.owasp>6.1.0</version.owasp>
```

It has been updated again
```suggestion
        <version.owasp>6.1.1</version.owasp>
```